zmon_worker_monitor/zmon_worker/notifications/opsgenie.py

```diff
@@ -126,12 +152,21 @@ def notify(cls,
             }
 
             if isinstance(custom_fields, dict):
-                data['details'].update(custom_fields)
+                if len(json.dumps(custom_fields)) >= MAX_CUSTOM_SIZE:
```

Comment:
This structure of code seems to repeated underneath. Maybe that's also a function?

Reply:
done